### ISSUE TITLE
Update to Lease/Agreement Maintenance

### ISSUE THREAD

**Issue by bowlenj** [ONRR/onrr.gov-site#1327]
Please update the contact list for Lease/Agreement Maintenance at the following webpage https://www.onrr.gov/about/contact.htm, with the attached pdf.

[Lease Agreement Maintenance Contacts_1-4-22.pdf](https://github.com/ONRR/onrr.gov-site/files/7809866/Lease.Agreement.Maintenance.Contacts_1-4-22.pdf)

**Comment by cthomasONRR** [comment]
@bowlenj This PDF failed the accessibility check. Could you please fix the document and resubmit? Thanks. 
![image](https://user-images.githubusercontent.com/72526208/148112590-6ba60f40-dcab-44ff-b82b-12ef7ad3009a.png)

**Comment by bowlenj** [comment]
I re-checked and I believe it should be good now, sorry about that!
[Lease Agreement Maintenance Contacts_1-4-22.pdf](https://github.com/ONRR/onrr.gov-site/files/7809997/Lease.Agreement.Maintenance.Contacts_1-4-22.pdf)

**Comment by bowlenj** [comment]
I re-submitted, I'm sorry about that!

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resources Revenue
303-231-3202
***@***.***


Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.




________________________________
From: Christine Thomas ***@***.***>
Sent: Tuesday, January 4, 2022 12:24 PM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Bowlen, Jessica G ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Update to Lease/Agreement Maintenance (Issue #1327)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@bowlenj<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbowlenj&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Ccb59ce8c4d7a4be09b9108d9cfb7e4df%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769210997424909%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qg0wSl57CPu0q9OkNX4rldNEcP0yqGAX0kCtE4BqSJE%3D&reserved=0> This PDF failed the accessibility check. Could you please fix the document and resubmit? Thanks.
[image]<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F72526208%2F148112590-6ba60f40-dcab-44ff-b82b-12ef7ad3009a.png&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Ccb59ce8c4d7a4be09b9108d9cfb7e4df%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769210997424909%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=4CXz9z8ufvWvGke%2Fhhqthg7ULhpefsrqGpDp9s4kQBA%3D&reserved=0>

—
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F1327%23issuecomment-1005105090&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Ccb59ce8c4d7a4be09b9108d9cfb7e4df%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769210997434874%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=JrnWMDkTdDSciv%2F%2Bc8IoQuKOXB9Ir35R41udCLyw%2BNg%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQ56VH2LT4QSSD57HTXNW2DUUNCNDANCNFSM5LIFR6WQ&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Ccb59ce8c4d7a4be09b9108d9cfb7e4df%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769210997434874%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=65B5P6srLSF%2BoNpj2WZy4d5kquDXLskiXsfyVS2m1Vo%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Ccb59ce8c4d7a4be09b9108d9cfb7e4df%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769210997444830%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Z6718jEgX5xkV%2F2sFazeap84P2JSTdB3KX7X7wU%2FZ3s%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Ccb59ce8c4d7a4be09b9108d9cfb7e4df%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769210997444830%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=8QWc7hyO78IWdM%2B01l4M8nugDkXdnGCG5q9M0Xpq910%3D&reserved=0>.

**Comment by cthomasONRR** [comment]
@bowlenj OK, thanks. This update is now on the dev site: http://2017onrr/about/contact.htm. Please review and give us your approval to make the change live.

**Comment by bowlenj** [comment]
Looks great, thank you!

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resources Revenue
303-231-3202
***@***.***


Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.




________________________________
From: Christine Thomas ***@***.***>
Sent: Tuesday, January 4, 2022 12:56 PM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Bowlen, Jessica G ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Update to Lease/Agreement Maintenance (Issue #1327)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@bowlenj<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fbowlenj&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Cf7c9fbac50b348eb03a508d9cfbc5f93%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769230237478832%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=PiNbgpabylWkLRRtHAZzYEflnqRdzbpDsQdRIgtkWfA%3D&reserved=0> OK, thanks. This update is now on the dev site: http://2017onrr/about/contact.htm. Please review and give us your approval to make the change live.

—
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FONRR%2Fonrr.gov-site%2Fissues%2F1327%23issuecomment-1005126267&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Cf7c9fbac50b348eb03a508d9cfbc5f93%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769230237488787%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=XDWdjohDXVtz8uqSB9Tc6NkbrYn%2Bdjyr2C6PNyDJhWw%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQ56VH7SIVEYDERE37FLOGLUUNGIBANCNFSM5LIFR6WQ&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Cf7c9fbac50b348eb03a508d9cfbc5f93%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769230237488787%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=mfrGeq3wj1oRnWSDdfAwDAKtItsyjT4eejhMTaPGJcs%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Cf7c9fbac50b348eb03a508d9cfbc5f93%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769230237498744%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qfn7jNELCfbDNkgo9X8epfq6yeXF3iQBglAHMmHkz%2BY%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cjessica.bowlen%40onrr.gov%7Cf7c9fbac50b348eb03a508d9cfbc5f93%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637769230237498744%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=iGmg1AeZlBTtK47tl8ODu530k8aq6F93Z7aZXsrK%2FIE%3D&reserved=0>.

**Comment by cthomasONRR** [comment]
@bowlenj This change is now live: https://www.onrr.gov/about/contact.htm. Closing this issue.
